cards.c

```diff
@@ -0,0 +1,167 @@
+//Matthew Kellerman- prints out five random cards with suits, I commented out the code I tried to use originally for the enums and structs.
+#include <stdio.h>
```

Comment:
this works, but the entire point of the assignment was to have you get this to work with enums and structs.